Add `hcloud_datacenter(s)` resources (#55)
hetznercloud/terraform-provider-hcloud: website/docs/d/datacenters.html.md

---
layout: "hcloud"
page_title: "Hetzner Cloud: hcloud_datacenters"
sidebar_current: "docs-hcloud-datasource-datacenters"
description: |-
  List all available Hetzner Cloud Datacenters.
---
# Data Source: hcloud_datacenters

> Provides list of available to user Hetzner Cloud Datacenters.

Reviewer: ```suggestion
Provides a list of available Hetzner Cloud Datacenters.
```

Author: Sorry for bad English, fixed :-)

This resource may be useful to create highly available infrastructure, distributed across several datacenters.

> ## Example Usage

Reviewer: Need a blank line between Line 10 and 11

Author: Ok, fixed
